dev/com.ibm.ws.jaxrs.2.0.client_fat/test-applications/hostnameverification/src/io/openliberty/jaxrs/client/fat/hostnameverification/servlet/HostnameVerificationClientTestServlet.java

```diff
@@ -0,0 +1,184 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.jaxrs.client.fat.hostnameverification.servlet;
+
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import javax.servlet.annotation.WebServlet;
+import javax.ws.rs.ProcessingException;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.junit.After;
+import org.junit.Test;
+
+import componenttest.annotation.SkipForRepeat;
+import componenttest.app.FATServlet;
+
+/*
+ * Test the different combinations of SSL Hostname Verification config in the server.xml
+ * and the JAX-RS "com.ibm.ws.jaxrs.client.disableCNCheck" client property.
+ * 
+ * Skip the tests for `restfulWS-3.0`+ since there is no current way to disable 
+ * hostname verificaiton with RESTEasy.
```

Comment:
Have you talked with James Perkins about this?   Wouldn't it be in RESTEasy's best interest to support this?

Reply:
I haven't, probably could. This is just to figure out what the heck works/doesn't currently.

Reply:
Honestly, I thought this was merged back in May, didn't realize it was still hanging out.